### DISCUSSION TITLE
What security level does age provide? (128bit / 256bit)

### DISCUSSION THREAD

**Discussion by adambvidex** [FiloSottile/age#423] [Q&A]
I want to make sure I'm understanding the age spec correctly. The *File Key* is 128bits. If the adversary knows the file key then they can decrypt your file, correct? So as a crypto-system we would say that age provides a 128bit security level?  (Despite the fact that it uses a 256bit cipher).

**Comment by FiloSottile** [comment]
Yes, age file keys are 128 bits. The use of 256 bit ciphers lets us not worry about multi-target scenarios without a nonce.

Note that the concept of "security levels" is [mostly outdated](https://www.imperialviolet.org/2014/05/25/strengthmatching.html).
